### PR TITLE
Merge release/19.3 (19.3-rc-2)

### PR DESCRIPTION
**Pull request by AliSoftware**
Merging the release branch after doing a new beta.

The new beta includes:
 * A fix in Jetpack Login (#16025 + #16034 + wordpress-mobile/WordPress-Login-Flow-Android#82)
 * Latest `strings.xml` from GlotPress — allowing us to have all the latest translations, ready to be tested earlier than last-day-of-sprint (see pxLjZ-6OF-p2#comment-18809)
 * Version bump in `version.properties`

### Conflicts Resolution

In `build.gradle`:
```
<<<<<<< merge/release-19.3-rc-2
    wordPressLoginVersion = '0.11.1'
=======
    wordPressLoginVersion = 'trunk-a229c42dc36badcc39f7be3d3ad5231bb031eee9'
>>>>>>> trunk
```

Resolved by **keeping neither**, and instead using `'trunk-6c8cd2f3c3873f3beca9309cae18c196df24eb84'`, because:
 - Login lib's `6c8cd2f3c3873f3beca9309cae18c196df24eb84` commit, which is currently the top commit on Login lib's `trunk`, corresponds to the merge of the merge of wordpress-mobile/WordPress-Login-Flow-Android#80 into Login lib's `trunk`
 - wordpress-mobile/WordPress-Login-Flow-Android#82 — which is the fix which was included in this `19.3-rc-2` beta as part of Login lib's `0.11.1` — was a cherry-pick of wordpress-mobile/WordPress-Login-Flow-Android#80 — the one which landed in Login lib's `trunk`. So pointing to `6c8cd2f3c3873f3beca9309cae18c196df24eb84` or pointing to `0.11.1` would both include the fix (via wordpress-mobile/WordPress-Login-Flow-Android#80 or wordpress-mobile/WordPress-Login-Flow-Android#80 respectively) in either case
 - But `6c8cd2f3c3873f3beca9309cae18c196df24eb84` sits on top of `a229c42dc36badcc39f7be3d3ad5231bb031eee9` (which WPAndroid's `trunk` was pointing to until now), so its content will still be included when pointing to the more recent `6c8cd2f3c3873f3beca9309cae18c196df24eb84` commit of Login lib.

Which is why it made more sense to me to point back to this one (and rejoin `trunk`) than pointing to `0.11.1` (which would otherwise not include PRs 76 and 78 from Login lib, which landed in Login lib's `trunk` since)

image

\cc @ashiagr

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ext {
     coroutinesVersion = '1.5.2'
     wordPressUtilsVersion = '2.4.0'
-    wordPressLoginVersion = 'trunk-a229c42dc36badcc39f7be3d3ad5231bb031eee9'
+    wordPressLoginVersion = 'trunk-6c8cd2f3c3873f3beca9309cae18c196df24eb84'
```
Comment (AliSoftware) [inline build.gradle:4]: See PR description for details/rationale about this choice during conflict resolution

Comment (ParaskP7) [reply build.gradle:4]: 👋 @AliSoftware !

I understand why `trunk-6c8cd2f3c3873f3beca9309cae18c196df24eb84` was used. 💯 

Question (❓): Irrespective of this PR, and merging that to `trunk`, shouldn't we now cut a new `Login` release, or update the existing `0.11.1` draft release, to make it point to the latest commit as well, just to keep things consistent. Wdyt? 🤔

**Review by ParaskP7 — APPROVED** [review]
LGTM! 💯

File: build.gradle (hunk 1 above)
Comment (AliSoftware) [reply build.gradle:4]: Yes definitely, good point 👍.

Both the `release/0.11.1` branch and `0.11.1` tag were created, but time to officially publish the GitHub release and merge login lib's `release/0.11.1` back to login lib's `trunk` indeed 👍
I will try to take care of that this afternoon

Comment (AliSoftware) [reply build.gradle:4]: Follow-up: https://github.com/wordpress-mobile/WordPress-Login-Flow-Android/pull/84

**Pull request merged**